js/msp/MSPHelper.js

```diff
@@ -593,8 +593,12 @@ var mspHelper = (function (gui) {
                     offset++;
                     RX_CONFIG.nrf24rx_id = data.getUint32(offset, true);
                     offset += 4;
+                    RX_CONFIG.nrf24rx_channel_count = data.getUint8(offset);
+                    offset += 1;
                 }
                 if (semver.gt(CONFIG.flightControllerVersion, "1.7.3")) {
+                    // unused byte for fpvCamAngleDegrees, for compatiblity with betaflight
```

Comment:
@fiam in RX_CONFIG frame?

Reply:
Yep, that's where ßF sends that byte. I guess their reasoning is that it affects input mixing.